mlflow/server/js/src/test-utils/Fixtures.js

```diff
@@ -1,7 +1,7 @@
 import { Experiment } from '../sdk/MlflowMessages';
 
 const createExperiment = ({
-  experiment_id = '0',
+  experiment_id = 0,
```

Comment:
I think currently experiment_id is a string in the proto, we need to adjust the propTypes to `string`.

Reply:
So it actually seems that `experiment_id` is an int still in the proto-generated JS messages (even though the protobuf itself has since been updated to make `experiment_id` a string). Thus the real correct long-term fix here is to update the proto-generated JS messages to reflect the current set of protobuf types. I'll revert this Fixtures.js change as it's incompatible with that future change (but will defer updating the protobuf types to a future PR as it seems a bit risky & I don't want to bundle it in with this other code).